doc/fuzzing.md

```diff
@@ -234,11 +234,13 @@ $ honggfuzz/honggfuzz --exit_upon_crash --quiet --timeout 4 -n 1 -Q \
 # OSS-Fuzz
 
 Bitcoin Core participates in Google's [OSS-Fuzz](https://github.com/google/oss-fuzz/tree/master/projects/bitcoin-core)
-program, which includes a dashboard of [publicly disclosed vulnerabilities](https://bugs.chromium.org/p/oss-fuzz/issues/list).
```

Comment:
Sorry for not catching this while reviewing pre-merge, but should this URL be https://bugs.chromium.org/p/oss-fuzz/issues/list?q=bitcoin-core&can=1 (open issues + closed issues) instead of https://bugs.chromium.org/p/oss-fuzz/issues/list?q=bitcoin-core (open issues)? :)

Reply:
There is a drop-down where you can select all issues